Sources/Commands/SwiftPackageTool.swift

```diff
@@ -496,10 +496,26 @@ extension SwiftPackageTool {
     struct DumpSymbolGraph: SwiftCommand {
         static let configuration = CommandConfiguration(
             abstract: "Dump Symbol Graph")
+		static let defaultMinimumAccessLevel = AccessLevel.public
 
         @OptionGroup(_hiddenFromHelp: true)
         var swiftOptions: SwiftToolOptions
-        
+
+		@Flag(help: "Pretty-print the output JSON.")
+		var prettyPrint = false
+
+		@Flag(help: "Skip members inherited through classes or default implementations.")
+		var skipSynthesizedMembers = false
+
+		@Option(help: "Include symbols with this access level or more. Possible values: \(AccessLevel.allValueStrings.joined(separator: " | "))")
+		var minimumAccessLevel = defaultMinimumAccessLevel
+
+		@Flag(help: "Skip emitting doc comments for members inherited through classes or default implementations.")
+		var skipInheritedDocs = false
+
+		@Flag(help: "Add symbols with SPI information to the symbol graph.")
+		var includeSpiSymbols = false
```

Comment:
Nit: it seems that it would be better to spell out "SPI" in uppercase, as it is an acronym, though I realize that this is definitely nitpicking.

Reply:
OK I can change that, no problem. Do you want me to add it in a separate commit or should i force push over the existing one?

Reply:
Either way works — I prefer to amend commits and force push to the PR branch, but other people push a series of fixups.  We squash commits when merging so it doesn't matter much either way for the final history.  Thanks!